### PR TITLE
Geolocation circles aruond gateways and minor fixes

### PR DESCRIPTION
**Pull request by miguelstorani**
image
image

### CODE REVIEW

File: src/components/utils/geolocation/geolocation.component.js [hunk 1]
```diff
@@ -3,44 +3,106 @@ import * as ReactLeaflet from "react-leaflet";
...
+      _.get(props, "location.longitude"),
+    ]);
+
+    if (
+      _.hasIn(props, "gatewaysLocations") &&
+      !_.isNull(props.gatewaysLocations) &&
+      !_.isUndefined(props.gatewaysLocations)
+    ) {
+      let gatewaysLocations = props.gatewaysLocations.map((gw) => {
+        const key = Object.keys(gw)[0];
```
Comment (cpfarher) [inline src/components/utils/geolocation/geolocation.component.js:46]: Object.keys(gw) always exists? and is an array with almost one element?

File: src/components/utils/geolocation/geolocation.component.js [hunk 2]
```diff
@@ -3,44 +3,106 @@ import * as ReactLeaflet from "react-leaflet";
...
+        position[1] + gwLocation[1],
+      ])
+      .map((e) => e / gwCount);
+  }
+
   return (
-    <div style={{width: "100%", height: "200px"}}>
-      {positionDefined && (
-        <Map center={position} zoom={10}>
+    <div style={{ width: "100%", height: "200px" }}>
```
Comment (cpfarher) [inline src/components/utils/geolocation/geolocation.component.js:81]: I think is better to change the height to 100% and manage the width and height outside the component

File: src/components/utils/geolocation/geolocation.component.js [hunk 3]
```diff
@@ -3,44 +3,106 @@ import * as ReactLeaflet from "react-leaflet";
...
+            <Marker position={position} opacity={1} icon={icon}></Marker>
+          )}
+
+          {gatewaysLocationsAvailable && !positionDefined &&
+            gatewaysLocations.map((gw) => {
+              return (
+                <Circle
+                  center={{ lat: gw.latitude, lng: gw.longitude }}
+                  fillColor="blue"
+                  radius={2000}
```
Comment (cpfarher) [inline src/components/utils/geolocation/geolocation.component.js:99]: you do not need to change this now, but maybe it is good to pass as props

**Review by cpfarher — CHANGES_REQUESTED** [review]
About all comments, the only that is mandatory to change I think is the height of the geolocation component. This component is used it in other sections of the web and we need to fit to the parent div.
About the other comments, if it will bring spent much time, it's ok to planified the changes for the future

File: src/components/utils/geolocation/geolocation.component.js (hunk 1 above)
Comment (miguelstorani) [reply src/components/utils/geolocation/geolocation.component.js:46]: Yes, it does always exists. It is an object with only one element.
image

**Review by cpfarher — APPROVED** [review]
LGTM!

**Pull request merged**